GroupRepresentationsForCAP/gap/SemisimpleCategoryMorphisms.gi

```diff
@@ -275,24 +275,11 @@ InstallMethod( Component,
     
 end );
 
-##
-InstallMethod( \*,
-               [ IsRingElement, IsSemisimpleCategoryMorphism ],
-               
-  function( r, morphism )
-    
-    return SemisimpleCategoryMorphism( Source( morphism ),
-             List( SemisimpleCategoryMorphismList( morphism ), elem -> [ r * elem[1], elem[2] ] ),
-             Range( morphism ) 
-           );
-    
-end );
-
-##
+## shouldn't this simplify delegate to r * morphism?
 InstallMethod( \*,
                [ IsSemisimpleCategoryMorphism, IsRingElement ],
                
-  function( r, morphism )
+  function( morphism, r )
```

Comment:
Please delete this function as well.

This code was written when there was no primitive `AddMultiplyWithElementOfCommutativeRingForMorphisms`.
Now, the implementation of the 2-sided *-operator is automatic.

Reply:
I deleted the method.